.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# How To Contribute
+To keep our repositories as uniform as possible, and work together effectively please consider the following:
+
+## Our Git Flow
```

Comment:
@ShaunEgan @SkippyZA @darrynten  - could really use some eyes on this doc, particularly

 - Out Git Flow
 - Naming Branches
 - [Not Done] Working with releases?